### ISSUE TITLE
Building service "customerservice" failed 

### ISSUE THREAD

**Issue by alexnowak** [eventuate-tram/eventuate-tram-examples-customers-and-orders#76]
I am getting the following error when running: 

./gradlew mysqlbinlogComposeBuild mysqlbinlogComposeUp

> Task :mysqlbinlogComposeBuild
Invalid interpolation format for "build" option in service "customerservice": "${EVENTUATE_JAVA_BASE_IMAGE_VERSION?}"

How must EVENTUATE_JAVA_BASE_IMAGE_VERSION be set?

Environment:
Linux Mint 20
Java: openjdk version "1.8.0_275"

**Comment by cer** [comment]
I'm guessing that your version of docker compose is old.

**Comment by alexnowak** [comment]
Yep, that was it! Upgraded docker-composer from version 1.18.0 to 1.27.4 and everything is working as expected. Thanks a bunch!
